server/borsen/io/feed_parsers/ritzau.py

```diff
@@ -23,12 +23,37 @@ class RitzauFeedParser(BaseRitzauFeedParser):
 
     def __init__(self):
         super().__init__()
-        self.default_mapping.update({"anpa_category": {"xpath": "sectionID/text()", "filter": self._category_filter}})
+        self.default_mapping.update(
+            {"anpa_category": {"xpath": "sectionID/text()", "filter": self.sectionID_category_filter}}
+        )
 
-    def _category_filter(self, category):
-        voc_categories = superdesk.get_resource_service("vocabularies").get_items(_id="categories")
+    def do_mapping(self, item, item_xml, setting_param_name=None, namespaces=None):
+        item = super().do_mapping(item, item_xml, setting_param_name, namespaces)
+
+        # Parse section to Category
+        data = item_xml.xpath("section/text()", namespaces=namespaces)
+        if data:
+            values = self.section_category_filter(data)
+            for val in values:
+                item["anpa_category"].append(val)
+
+        # Add Default Category:
+        item["anpa_category"].append(
+            {
+                "name": "Generelt",
```

Comment:
would rather read this data from vocabulary, using the `qcode` to select the value